core/tests/protractor_desktop/adminMiscTab.js

```diff
@@ -55,7 +55,7 @@ describe('Admin misc test tab', function() {
     await adminPage.getMiscTab();
     await adminPage.uploadTopicSimilarities(
       '../data/sample_topic_similarities.csv', true);
-    allowedErrors.push('encode', 'Object', 'resource');
+    await allowedErrors.push('encode', 'Object', 'resource');
```

Comment:
Do we need to `await` this? I think you're just pushing an element onto a list, which should be synchronous I think